### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bhepple**
Add temporary fedora installation instructions

### CODE REVIEW

**Comment by jtheoof** [comment]
Thanks @bhepple  do you mind using [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) guideline for your commit? It will help generate the CHANGELOG.md automatically.

I am thinking something like: `docs(readme): add temporary fedora install`. Thanks for your PR !

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ The following lines can be used as swappy's default:
 ## Installation
 
 - [Arch Linux](https://aur.archlinux.org/packages/swappy-git)
+- [fedora 31/32](https://copr.fedorainfracloud.org/coprs/wef/swappy)
```
Comment (jtheoof) [inline README.md:98]: I would prefer `Fedora` here instead of `fedora`, I see the capital Fedora is used on their [website](https://getfedora.org/)

Comment (bhepple) [reply README.md]: OK, done.

**Comment by bhepple** [comment]
Hi Jeremy,

I'm using fontawesome-fonts-4.7.0-7 on fedora-31 and it seems to be
just fine however, the README specifies 'Font Awesome 5' so am I
missing something?

I need to confirm this before fedora will accept the package.

Thanks

Bob

**Comment by jtheoof** [comment]
Hi Bob,

Considering the icons that I currently use, swappy should work with FA 4. But if I need to add more tools (and so icons) in the future, I will pick from FA 5, which has a lot more than FA 4 so it might not work in the future. Therefore I would still recommend using FA >=5, but it's technically OK to have FA >= 4 at the moment.

Hope this helps.

**Comment by bhepple** [comment]
Hi Jeremy,

Yes, thanks, that explains it just fine. I'll put a note to that effect in
the spec so that we keep an eye on the issue.

If there's some way you can let us know if a new release _requires_ FA5
then we can avoid problems eg by doing a fedora-specific patch.

Of course, Fedora may have FA5 by then anyway!

Cheers


Bob

**Comment by bhepple** [comment]
Hi Jeremy,

We need to gpg verify the tarballs as we download them but I'm having no
luck discovering your keyring/public key declared as 6A6B35DBE9442683 on
github.

$ gpg2 --recv-key 0x6A6B35DBE9442683
gpg: keyserver receive failed: No data

Have you published it anywhere?

Thanks


Bob

**Comment by jtheoof** [comment]
No it's not published anywhere as of now. On which gpg server should I publish it? (I don't have much experience with gpg keys yet, I just used it to sign the commit / tag).

**Comment by bhepple** [comment]
Probably (I'm no expert) this would work:

$ gpg2 --send-keys 0x6A6B35DBE9442683

I believe that it will only send the public parts ...

Most of the keyservers out there sync their databases so it should not
matter which one it goes to in the long run. There is a --keyserver
<server> option if you want to be specific. For example, I get this
one:

$ gpg2 --verbose --search-key 0x587979573442684E
gpg: data source: https://192.146.137.141:443
(1)    Steve McIntyre <93sam@debian.org>
    Steve McIntyre <steve@einval.com>
    Steve McIntyre <stevem@chiark.greenend.org.uk>
      4096 bit RSA key 587979573442684E, created: 2009-05-09

... it might help me if you use the same one (or if you let me know
which one you use) so I don't have to wait for the sync (although I
have no right to insist, nor would it be crypto-encouraged for me to
do so!)

Cheers


Bob

**Comment by jtheoof** [comment]
Thanks Bob, yes I checked on my side [on Arch](https://wiki.archlinux.org/index.php/GnuPG), you're right the keys are shared across servers. I will publish it end of next week, I'm out on vacation at the moment. Sorry for the delay.

**Comment by jtheoof** [comment]
Hi @bhepple, you should see my key now.

**Comment by bhepple** [comment]
Not yet:

$ gpg2 --verbose --search-key 0x6A6B35DBE9442683
gpg: data source: https://209.244.105.201:443
gpg: key "0x6A6B35DBE9442683" not found on keyserver
gpg: keyserver search failed: Not found

I'll try again tomorrow

BTW swappy-1.1.0 is about 2 days from release in fedora-31/32

Cheers

Bob

**Pull request merged**